scripts/DB/sample_data.sql

```diff
@@ -52,3 +52,18 @@ INSERT INTO "GroupMemberships" ("UserId", "GroupId") VALUES
 ( 7,  14),
 ( 7,   1),
 ( 5,  14);
+
+INSERT INTO "Projects" ("Name", "TypeId", "Description", "OwnerId", "GroupId", "OrganizationId", "Language","Private", "DateCreated", "DateUpdated", "DateArchived", "AllowDownloads", "AutomaticBuilds") VALUES
```

Comment:
You shouldn't provide DateCreated, DateUpdated, or DateArchived.  First 2 will be automatically populated and the 3rd can be left null.

Reply:
Nevermind.  Since these are just SQL statements, then it won't go through our code to set the datetime.